app/models/like.rb

```diff
@@ -1,8 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
-  belongs_to :post, counter_cache: :likes_counter
+  belongs_to :post
 
-  def update_counters
-    update(likes_counter: likes.count)
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment(:likes_counter)
```

Comment:
- [ OPTIONAL ] When you do the `after_save` you should make the update private example: 
 ```ruby
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```